### PR TITLE
Add info modal for unavailable email templates

### PR DESCRIPTION
**Pull request by JeethJJ**
Add info message to alert when selected locale of an email template from sub-org which is not available.

Design : 
Screenshot 2024-12-16 at 10 21 29

### CODE REVIEW

File: apps/console/src/extensions/i18n/resources/en-US/extensions.ts [hunk 1]
```diff
@@ -2967,6 +2967,11 @@ export const extensions: Extensions = {
                 replicateContent: {
                     header: "Replicate content?",
                     message: "Seems like you don't have any content for this locale. Do you need to populate the previous locale's content here as a quick start?"
+                },
+                updateFromRootOrg: {
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/i18n/resources/en-US/extensions.ts:2971]: Any particular reason we have used `rootorg` phrase in i18n key? based on the context, I think the issue being addressed has nothing to do with organization type, has it?

**Review by pavinduLakshan — COMMENTED** [review]
Let's fix the lint issues as well

File: apps/console/src/extensions/i18n/resources/en-US/extensions.ts (hunk 1 above)
Comment (JeethJJ) [reply apps/console/src/extensions/i18n/resources/en-US/extensions.ts:2971]: The issue has context related to root org. The email template for different locale can only be updated from the root org and only viewable from the sub org.

**Comment by JeethJJ** [comment]
> Let's fix the lint issues as well

Addressed with https://github.com/wso2/identity-apps/pull/7209/commits/0ecfe2ec0058f4647f92bd0afcc1c342053dd4f4

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**